### ISSUE TITLE
`--testing` flag does not prevent overwriting of result files

### ISSUE THREAD

**Issue by milindsmart** [blue-marble/gridpath#765]
`gridpath_run` has the `--testing` flag which should suppress saving of results https://github.com/blue-marble/gridpath/blob/8a8042b667ff19aea3f283bd41f56e484af5ce92/gridpath/common_functions.py#L173-L174

However, scenario results are saved and the test results for verification are overwritten. `run_scenario.py` does not seem to have any branching logic for this flag.

**Comment by anamileva** [comment]
Yep, this should have been removed from the help menu as it's not hooked up anywhere at this point. I see there are still some references to "testing mode" in `run_scenario.py`, which should also be clarified.

**Comment by milindsmart** [comment]
I feel this is still useful functionality to have, FWIW. Currently the example test values to be compared and checked are within the test script itself, maybe it can read from the result files?
